staging/src/k8s.io/apiserver/pkg/admission/attributes.go

```diff
@@ -42,12 +43,22 @@ type attributesRecord struct {
 
 	// other elements are always accessed in single goroutine.
 	// But ValidatingAdmissionWebhook add annotations concurrently.
-	annotations     map[string]string
+	annotations     map[string]annotation
 	annotationsLock sync.RWMutex
 
 	reinvocationContext ReinvocationContext
 }
 
+// annotation holds a pair of intended audit level and annotation string
+type annotation struct {
```

Comment:
would remove drop the godoc comments actually. It's just a pair with obvious meaning.